source/data-api/custom-endpoints.txt

```diff
@@ -335,6 +345,27 @@ scheme for each custom endpoint by configuring the endpoint function.
 
       To learn how to sign your requests, see :ref:`Authorize the Request <endpoint-authorize>`.
 
+Custom Authorization Schemes
+++++++++++++++++++++++++++++
+
+You can define a custom authorization :ref:`expression <expressions>` to
+determine if an incoming authenticated request is allowed to run. The
+expression is evaluated for each request and must evaluate to ``true``
+to allow the request. If the expression evaluates to ``false``, the
+request is not authorized and fails with an error. Requests that fail
+authorization are not counted toward your App's billed usage.
```

Comment:
would be good to link to the [`%%request` expansion](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/data-api-can-evaluate/rules/expressions/#request-expansions) somewhere in this description.